portal/app/core/templates/partials/loads.html

```diff
@@ -1,43 +1,73 @@
-{% extends "resources.html" %}
+{% extends "dashboard.html" %}
 {% load static %}
 
-{% block resources-styles %}
-{% endblock resources-styles %}
+{% block dashboard-styles %}
+{% endblock dashboard-styles %}
 
 {% block dashboard-resources %}
 <main role="main" class="hidden-xs">
     Home Circuit and Plug Loads
     <div class="container">
       <div class="row hidden-xs loads-container">
-        <div class="col-sm-4 cell" align="center">
+        <div class="col-sm-4" align="center">
             Living Room: 200W
+        </div>
+        <div class="col-sm-4" align="center">
+            Bedrooms: 203W
+        </div>
+        <div class="col-sm-4" align="center">
+            Kitchen: 504W
+        </div>
+
+        <div class="col-sm-4 cell" align="center">
             <img src="https://static.thenounproject.com/png/81671-200.png" height="120">
         </div>
         <div class="col-sm-4 cell" align="center">
-            Bedrooms: 203W
             <img src="https://static.thenounproject.com/png/81673-200.png" height="120">
         </div>
         <div class="col-sm-4 cell" align="center">
-            Kitchen: 504W
             <img src="https://static.thenounproject.com/png/81669-200.png" height="120">
         </div>
-        <div class="col-sm-4 cell" align="center">
+
+
+
+
+
```

Comment:
Please remove all the unnecessary whitespace

Reply:
done